mix.exs

```diff
@@ -4,7 +4,7 @@ defmodule GRPC.Mixfile do
   def project do
     [
       app: :grpc,
-      version: "0.6.4",
+      version: "0.7",
```

Comment:
I know that we're closer to _romantic_ then _semantic_ versioning, but the behaviour didn't change at all, so what about
```suggestion
      version: "0.6.5",
```
?